xwiki-commons-core/xwiki-commons-velocity/src/main/java/org/xwiki/velocity/tools/URLTool.java

```diff
@@ -58,4 +61,55 @@
         }
         return queryParams;
     }
+
+    /**
+     * Convert a map into a query string.
+     * 
+     * @param map the map to be converted
+     * @return the converted query string
+     * @since 10.4
+     */
+
+    public String getQueryString(Map<String, String> map)
```

Comment:
`serializeQueryString` ?

Reply:
You also need to document if the user needs to encore or not the key and value or if it's done by the API.